### PR TITLE
Feat : Added Modal

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

I have added Modal component to my UI Library.
Can you please review it
Link : https://monumental-pie-c9545b.netlify.app/components/modal.html

The site is not responsive yet

Thanks

### CODE REVIEW

**Review by syed-shaida-hussain — APPROVED** [review]
Neatly done , no changes required.

File: utility.css [hunk 1]
```diff
@@ -209,3 +213,7 @@ img{
 .secondary-text-colour{
     color:var(--secondary-color)
 }
+
+.light-shadow{
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```
Comment (ranitasahaa1230) [inline utility.css]: Declare this in your root variable

File: Components/modal.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "/main.css">
```
Comment (ranitasahaa1230) [inline Components/modal.html:7]: You can add a description in your meta tag like author details, page description etc.

**Review by ranitasahaa1230 — APPROVED** [review]
Code looks clean
Just a suggestion, you can add another live demo modal with the help of JS. Like close the pop-up, click on close icon etc.

File: Components/modal.html (hunk 2 above)
Comment (shlok30) [reply Components/modal.html:7]: Understood

**Pull request merged**